lib/rabbit/rabbit.ex

```diff
@@ -33,16 +26,12 @@ defmodule PhxPlatformUtils.Mqtt do
         if String.match?(ip_string_or_host, ~r/^((25[0-5]|(2[0-4]|1[0-9]|[1-9]|)[0-9])(\.(?!$)|$)){4}$/) do
           ip_string_or_host |> String.to_charlist()
```

Comment:
does it need a charlist?

Reply:
if it's an underlying erlang lib expecting a string, it would need a charlist, otherwise should be a string.

Reply:
I'm not sure, I'll double check.